components/Table/Table.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <div>
+    <table>
+      <thead>
+        <tr>
+          <th v-for="(head,i) in headers" :key="i">
+            <div :style="align(head.align)">{{ head.text}}</div>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr v-for="(row, rowIndex) in items" :key="rowIndex">
+          <td v-for="(header,i) in headers" :key="i">
+            <div :style="align(header.align)">
+              <a v-if="header.link" :href="row[header.value]" target="_blank">
+                <img :src="src(header.icon)" :srcset="srcset(header.icon)" />
```

Comment:
why is there an image for every link? this component should be generic.

Reply:
I will change this like to slot. Thanks.